dist/Dockerfile.deploy/Dockerfile

```diff
@@ -23,14 +23,15 @@ RUN \
   find $HOME/. -type d -exec chmod 777 {} \; && \
   find $HOME/. -type f -exec chmod ugo+rw {} \;
 
+WORKDIR /build/cincinnati
 COPY . .
 # copy git information for built crate
 COPY .git/ ./.git/
 
 RUN cargo build --release && \
     mkdir -p /opt/cincinnati/bin && \
-    cp -rvf target/release/graph-builder /opt/cincinnati/bin && \
-    cp -rvf target/release/policy-engine /opt/cincinnati/bin
+    cp -rvf /build/cincinnati/target/release/graph-builder /opt/cincinnati/bin && \
+    cp -rvf /build/cincinnati/target/release/policy-engine /opt/cincinnati/bin
```

Comment:
do we need this change?  I'd expect `WORKDIR` to be the current working directory for subsequent `RUN`